src/module_type_translator.ts

```diff
@@ -219,7 +219,7 @@ export class ModuleTypeTranslator {
     if (this.host.untyped) return;
     // Already imported? Do not emit a duplicate forward declare.
     if (this.forwardDeclaredModules.has(moduleSymbol)) return;
-    if (this.host.typeBlackListPaths && this.host.typeBlackListPaths.has(importPath)) {
+    if (typeTranslator.isBlacklisted(this.host.typeBlackListPaths, moduleSymbol)) {
       return;  // Do not emit goog.forwardDeclare or goog.require for blacklisted paths.
```

Comment:
Change the comment to `// ... symbols.`?

Reply:
Done.